piker/data/_sampling.py

```diff
@@ -229,16 +231,71 @@ async def sample_and_broadcast(
             # thus other consumers still attached.
             subs = bus._subscribers[sym.lower()]
 
-            for ctx in subs:
-                # print(f'sub is {ctx.chan.uid}')
-                try:
-                    await ctx.send_yield({sym: quote})
-                except (
-                    trio.BrokenResourceError,
-                    trio.ClosedResourceError
-                ):
-                    # XXX: do we need to deregister here
-                    # if it's done in the fee bus code?
-                    # so far seems like no since this should all
-                    # be single-threaded.
-                    log.error(f'{ctx.chan.uid} dropped connection')
+            for (stream, tick_throttle) in subs:
+
+                if tick_throttle:
```

Comment:
I feel like we could just make the mem chan look like the `MsgStream` here and avoid the flag check, but not gonna stress about it right now.